main/test/ca/uwaterloo/flix/language/phase/TestStratifier.scala

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2017 Jason Mittertreiner
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ca.uwaterloo.flix.language.phase
+
+import ca.uwaterloo.flix.TestUtils
+import ca.uwaterloo.flix.api.Flix
+import ca.uwaterloo.flix.language.errors.StratificationError
+import ca.uwaterloo.flix.runtime.Model
+import ca.uwaterloo.flix.util.Options
+import org.scalatest.FunSuite
+
+class TestStratifier extends FunSuite with TestUtils {
```

Comment:
I would like to see all the *negative* test cases here, but all the positive test cases af Flix code in some appropriate place like `flix/language/feature/StratifiedNegation.flix`